formwidgets/RevisionHistory.php

```diff
@@ -51,8 +56,12 @@ public function onRevertHistory()
         $section = $modelClass::find($this->model->id);
 
         $revision = Revision::find(input('revision_id'));
+        $revisions = Revision::where('user_id', $revision->user_id)->where('created_at', $revision->created_at)->get();
```

Comment:
Why was this changed? Because now it reverts all changes by that user not a specific that user selected.

Reply:
The idea is that a user selects a group of changes that was made at the same time. These changes will then all rollback at the same time.

Reply:
Then it should be made as separate option for multiple and single one.

Reply:
I added checkboxes before each revision to select the ones you want to restore.